### PR TITLE
gnome-secrets: update to 7.0; New package: python3-zxcvbn-4.4.28

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

**Comment by cinerea0** [comment]
`python3-zxcvbn` is a new dependency of `gnome-secrets`. The other two changes are due to `gnome-secrets` requiring functionality from `pykeepass` that's only available in newer releases.

File: srcpkgs/python3-construct/template [hunk 1]
```diff
@@ -10,7 +10,7 @@ maintainer="Enno Boland <gottox@voidlinux.org>"
 license="MIT"
 homepage="https://construct.readthedocs.org/"
 distfiles="${PYPI_SITE}/c/construct/construct-${version}.tar.gz"
-checksum=3c130a67f05610557dd1b27e062a7165f92b517f9ed50db880ebe068696fd712
+checksum=7b2a3fd8e5f597a5aa1d614c3bd516fa065db01704c72a1efaaeec6ef23d8b45
 
 do_check() {
```
Comment (classabbyamp) [inline srcpkgs/python3-construct/template:15]: remove `do_check` and either add the relevant `checkdepends` or mark `make_check=no`

Comment (cinerea0) [reply srcpkgs/python3-construct/template]:
```
=> python3-construct-2.10.68_1: running do_check ...
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
running egg_info
writing construct.egg-info/PKG-INFO
writing dependency_links to construct.egg-info/dependency_links.txt
writing requirements to construct.egg-info/requires.txt
writing top-level names to construct.egg-info/top_level.txt
reading manifest file 'construct.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
writing manifest file 'construct.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "/builddir/python3-construct-2.10.68/setup.py", line 5, in <module>
    setup(
  File "/usr/lib/python3.11/site-packages/setuptools/__init__.py", line 87, in setup
    return distutils.core.setup(**attrs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/setuptools/_distutils/core.py", line 185, in setup
    return run_commands(dist)
           ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/setuptools/_distutils/core.py", line 201, in run_commands
    dist.run_commands()
  File "/usr/lib/python3.11/site-packages/setuptools/_distutils/dist.py", line 968, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.11/site-packages/setuptools/dist.py", line 1217, in run_command
    super().run_command(command)
  File "/usr/lib/python3.11/site-packages/setuptools/_distutils/dist.py", line 987, in run_command
    cmd_obj.run()
  File "/usr/lib/python3.11/site-packages/setuptools/command/test.py", line 224, in run
    self.run_tests()
  File "/usr/lib/python3.11/site-packages/setuptools/command/test.py", line 227, in run_tests
    test = unittest.main(
           ^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/main.py", line 101, in __init__
    self.parseArgs(argv)
  File "/usr/lib/python3.11/unittest/main.py", line 127, in parseArgs
    self._do_discovery(argv[2:])
  File "/usr/lib/python3.11/unittest/main.py", line 247, in _do_discovery
    self.createTests(from_discovery=True, Loader=Loader)
  File "/usr/lib/python3.11/unittest/main.py", line 157, in createTests
    self.test = loader.discover(self.start, self.pattern, self.top)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 322, in discover
    tests = list(self._find_tests(start_dir, pattern))
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 377, in _find_tests
    tests, should_recurse = self._find_test_path(full_path, pattern)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 453, in _find_test_path
    tests = self.loadTestsFromModule(package, pattern=pattern)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/site-packages/setuptools/command/test.py", line 57, in loadTestsFromModule
    tests.append(self.loadTestsFromName(submodule))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/unittest/loader.py", line 214, in loadTestsFromName
    raise TypeError("don't know how to make test from: %s" % obj)
                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~
TypeError: not all arguments converted during string formatting
=> ERROR: python3-construct-2.10.68_1: do_check: '${make_check_pre} python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
```
Not really sure what to make of this, maybe just a basic incompatibility. I added `make_check=no`.

Comment (classabbyamp) [reply srcpkgs/python3-construct/template]: i think you need pytest

Comment (cinerea0) [reply srcpkgs/python3-construct/template]:
```
=> python3-construct-2.10.68_1: running do_check ...
============================= test session starts ==============================
platform linux -- Python 3.11.0, pytest-7.1.3, pluggy-1.0.0
rootdir: /builddir/python3-construct-2.10.68
collected 0 items

============================ no tests ran in 0.01s =============================
=> ERROR: python3-construct-2.10.68_1: do_check: 'PYTHONPATH="$(cd build/lib* && pwd)" ${make_check_pre} python3 -m pytest ${testjobs} ${make_check_args} ${make_check_target}' exited with 5
=> ERROR:   in do_check() at common/build-style/python3-module.sh:15
```
Maybe there just aren't any tests?

Comment (classabbyamp) [reply srcpkgs/python3-construct/template]: there are definitely tests in there: https://github.com/construct/construct/blob/master/tests/test_core.py

Comment (cinerea0) [reply srcpkgs/python3-construct/template]: There are tests, but not in the pypi tarball. I switched the source to github and the tests are there, but they require [cloudpickle](https://pypi.org/project/cloudpickle/) which we don't package. `python3-joblib` includes files related to cloudpickle, but it doesn't satisfy the requirement.

Comment (classabbyamp) [reply srcpkgs/python3-construct/template]: ok then it's reasonable to disable i think

**Comment by cinerea0** [comment]
> commit message should include the version

Fixed in the latest push.

**Pull request merged**